certbot/compat/os.py

```diff
@@ -0,0 +1,20 @@
+"""
+This compat modules extends native capabilities of core os module to handle correctly platform
+specific operations (eg. chown, chmod, geuid).
+This module is intended to replace standard os module throughout certbot projects (except acme).
+"""
+from __future__ import absolute_import
+
+# Expose everything from standard os package to make current package a complete replacement of os.
+from os import *  # type: ignore  # pylint: disable=wildcard-import,unused-wildcard-import,redefined-builtin,os-module-forbidden
```

Comment:
Is there a reason to do this when we also have the loop below?

Reply:
Initially it was the way I used to import all os module content into this module. It makes pylint, mypy, IDEs like pycharm very happy, because it allows them to understand, through this static import, that all public members of os are also here.

However, it appears that the `__all__` of `os` is incomplete in some python version (3.5 if I recall correctly), making this static import insufficient in this case.

So I added a dynamic import to cover all the cases, and kept this one to continue benefiting of the advantages of static import I described above. 

I think all of this worths a comment ^^

Reply:
Comment added.